website/content/docs/configuration/server.mdx

```diff
@@ -135,15 +135,33 @@ server {
   a tradeoff as it lowers failure detection time of nodes at the tradeoff of
   false positives and increased load on the leader.
 
-- `failover_heartbeat_ttl` `(string: "5m")` - Specifies the TTL applied to
-	heartbeats after a new leader is elected, since we no longer know the status
-	of all the heartbeats. This is specified using a label suffix like "30s" or
-	"1h".
+- `failover_heartbeat_ttl` `(string: "5m")` - Specifies the initial TTL applied to
+  Client heartbeats after a new leader is elected. This TTL must be high enough
+  such that a Client previously connected to a now-unresponsive server will be
+  able to find a new healthy server through which to send heartbeats to the new
+  leader. The strict minimum value depends on the number of nodes in the cluster,
+  the value of `max_heartbeats_per_second`, `min_heartbeat_ttl`, and `heartbeat_grace`.
+  As the number of nodes in the cluster increases, the longer the TTL each client
+  is assigned. This value is uniformly randomized between the values computed by
+  these two formula:
 
-  ~> Lowering the `failover_heartbeat_ttl` is a tradeoff as it lowers failure
+```
+<min ttl> = max(((<num nodes> / <max_heartbeats_per_second>) + <heartbeat_grace>), <min_heartbeat_ttl>)
+<max ttl> = max(((2 * (<num nodes> / <max_heartbeats_per_second>)) + <heartbeat_grace>), <min_heartbeat_ttl>)
+```
+
+  The default value of `"5m"` is fine for small and medium sized clusters of fewer
+  than approximately 5k nodes - the value of `failover_heartbeat_ttl` must be
+  higher than `<max ttl>`, ideally with some extra padding for clusters growing
+  over time. If `<max ttl>` exceeds the failover heartbeat ttl, the cluster may
+  experience Clients being marked down unexpectedly after leader transitions - in
+  particular by Clients that were connected to a leader that has suddenly gone
+  offline.
+
+  Lowering the `failover_heartbeat_ttl` is a tradeoff as it lowers failure
   detection time of nodes at the tradeoff of false positives. False positives
-  could cause all clients to stop their allocations if a leadership transition
-  lasts longer than `heartbeat_grace + failover_heartbeat_ttl`.
+  will cause clients to be marked as node status down, and their allocations will
+  be marked as lost.
```

Comment:
Nitpick as written: capitalize Client to be consistent and put `down` in backticks

...but I think we can do better on this whole paragraph. It's not wrong per se, but it lacks the leader election context that this particular setting uniquely applies to.

Perhaps:

> Lowering the `failover_heartbeat_ttl` gives Clients less time to potentially failover from a dead Server to a live one in the case of a Server crash causing a leader election. The failover TTL must be higher than the max heartbeat TTL as determined above with additional buffer to account for network latency, timeouts, and potentially rediscovering Server addresses via Consul.
> Raising the `failover_heartbeat_ttl` increases the time in which a down Client will *not* have its work rescheduled. This can be a particular problem if the loss of a datacenter causes a Server and number of Clients to be lost. The remaining quorum of Servers and Clients will not reschedule the down datacenter's work until this failover TTL has elapsed.

...but idk... I might be getting too wordy.